types.ts

```diff
@@ -433,6 +433,20 @@ export interface OAuth2Authentication extends BaseAuthentication {
    * See https://datatracker.ietf.org/doc/html/rfc7636 for more details.
    */
   useProofKeyForCodeExchange?: boolean;
+
+  /**
+   * In rare cases, OAuth providers may want the permission scopes in a different query parameter
+   * than `scope`. You can use the `additionalParams` above to accomplish that. In even rarer
+   * cases, some of those OAuth providers sometimes want you to also *not* send the normal scope
+   * param (which this parameter enables).
+   */
+  suppressScopeParam?: boolean;
```

Comment:
Would it work to make this `scopeParamName` instead and have it override what we use for the scope param? For Slack, we then set `scopeParamName: 'user_scope'` and then remove the hack where we currently pass the scopes as additionalParams.

Reply:
Great idea! Done, and tested with `coda auth` on an updated https://github.com/coda/packs/pull/3103